src/FakerDotNet/Faker.cs

```diff
@@ -16,6 +16,7 @@ public static class Faker
         public static INumberFaker Number { get; } = Container.Number;
         public static IRandomFaker Random { get; } = Container.Random;
         public static ITimeFaker Time { get; } = Container.Time;
+        public static IFriendsFaker Friends { get; } = Container.Friends;
```

Comment:
These properties are supposed to be in alphabetical order. It makes it easier for new contributors to know where to add the new properties into the `Faker` and `FakerContainer` classes :+1: